amp-pwa-reader/src/js/DragObserver.js

```diff
@@ -49,18 +54,34 @@ class DragObserver extends Evented {
 
   _start (event) {
 
-    // store event for re-use
-    this.eventDown = event;
+    if (event.pageX) {
+      this.eventDown = event;
+    } else {
+      this.eventDown = event.targetTouches[0];
```

Comment:
as noted, this is probably not needed as pointer-events seem to include pageX/pageY on the top level object nowadays.